koku/api/provider/provider_manager.py

```diff
@@ -184,6 +185,12 @@ def remove(self, request, customer_remove_context=False):
             raise ProviderManagerError(err_msg)
 
 
+@receiver(post_save, sender=Provider)
+def provider_post_save_callback(sender, instance, created, **kwargs):
+    """Kickoff report download on Provider.save."""
+    check_report_updates.delay(provider_uuid=instance.uuid)
```

Comment:
Do you know if you have information about the request here? If so you could check the request.method and if it's patch *not* call the task?

Reply:
Looks like there is the possibility of a request_finished signal that would likely have info about the request + method.

https://docs.djangoproject.com/en/3.0/ref/signals/#request-finished